tutorials/text/annotations.py

```diff
@@ -92,6 +92,30 @@
 
 # %%
 #
+# Annotating an Artist
+# ~~~~~~~~~~~~~~~~~~~~
+#
+# Annotations can be positioned relative to an `.Artist` instance by passing
+# that Artist in as *xycoords*. Then *xy* is interpreted as a fraction of the
+# Artist's bounding box.
+
+import matplotlib.patches as mpatches
+
+fig, ax = plt.subplots(figsize=(3, 3))
+arr = mpatches.FancyArrowPatch((.25, .25), (.75, .75),
+                               arrowstyle='->,head_width=.15', mutation_scale=20)
+ax.add_patch(arr)
+ax.annotate("label", (.25, .25), xycoords=arr, ha='center', va='center')
+ax.set(xlim=(0, 1), ylim=(0, 1))
+
+# %%
+# Here the annotation is placed at position (.25,.25) relative to the arrow's
```

Comment:
```suggestion
ax.annotate("label", (.35, .25), xycoords=arr, ha='center', va='center')
ax.set(xlim=(0, 1), ylim=(0, 1))

# %%
# Here the annotation is placed at position (.35,.25) relative to the arrow's
```

You use `(.25, .25)` above so it's a bit confusing to also use it for the annotation location.  Suggest they be different numbers....